indigo_app/static/stylesheets/_treeview.scss

```diff
@@ -0,0 +1,23 @@
+.treelist-wrapper {
+  ul {
+    padding-left: 0;
+  }
+  li {
+    display: flex;
+    align-items: flex-start;
+  }
+
+  .spacer {
+    width: 22px;
+  }
+
+  input[type="checkbox"]  {
+    margin-top: 3px;
+    width: 12px;
```

Comment:
This isn't a good idea. Checkboxes are weird: the widgets differ between platforms so setting the actual widget size is asking for trouble (as you discovered with Firefox on linux).

Rather than setting the widget width, if you put it inside a wrapping div (eg `.marker` to match `.content`) then the div size can be set consistently and the div content is unstyled. That would also mean you don't need to differentiate between having content and having a spacer.

Reply:
refactored